src/core/src/command/mod.rs

```diff
@@ -74,10 +59,9 @@ impl<'a, B: Backend, C> CommandBuffer<'a, B, C> {
     /// The command buffer will be consumed and can't be modified further.
     /// The command pool must be reset to able to re-record commands.
     pub fn finish(mut self) -> Submit<B, C> {
-        let submit = self.raw.finish();
+        self.raw.finish();
```

Comment:
we might also want to call `self.raw.finish()` in the destructor

Reply:
Would you mind elaborating?